docs/source/index.rst

```diff
@@ -1,119 +1,135 @@
-    have written about Dissect in :doc:`/resources/dissect-in-action`.
 
+Usage Example
+-------------
+
+One of the most prominent tools that Dissect offers is called *target-query*
+With this simple one-liner you can extract all user accounts from an image: 
 
-Getting Started
----------------
+.. code-block:: console
 
-The easiest way to get started is to install the latest version of all Dissect projects from
-`PyPI <https://pypi.python.org/pypi/dissect>`_:
+    $ target-query pc.img -f users
+    
+    <windows/user ... name='systemprofile' ...>
+    <windows/user ... name='LocalService' ...>
+    <windows/user ... name='NetworkService' ...>
+    <windows/user ... name='Mr. Evil' ...>
+    
+To see what other useful artefacts you can query use ``-l``:
 
 .. code-block:: console
 
-    $ pip install dissect
+    $ target-query pc.img -l
 
-It is recommended that you use a `virtual environment <https://docs.python.org/3/tutorial/venv.html>`_.
+This will list all Dissect functions and artefacts that are available for this image.
+To list **all** functions and artefacts, regardless of the image, just leave out the image argument!
 
-.. note ::
-    .. include:: /versions.rst
+Try the :doc:`/tutorial` to get started!
 
-To quickly get familiar with Dissect, read on to :doc:`/usage/introduction`. If you're interested in what makes it
-tick, continue reading at :doc:`/overview/index`.
+Key features
+------------
 
-Installing individual projects
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Dissect is a powerful artefact extractor and parser that saves you a lot of time.
+With Dissect you can:
 
-Dissect is a collection of libraries and tools, so every Dissect project is individually installable. You'll find that
-installing the ``dissect`` package from PyPI will install all available ``dissect.*`` packages for you! If, however,
-you just want to work with the NTFS and registry parsers in your own scripts, you can opt to only install or depend
-on those specific projects:
+* Quickly **extract artefacts** from any source (like IMG, EWF, Kape, DD, VDI, PVM)
+* Access artefacts from almost any OS (like Windows, macOS, Linux/Unix, ESXi)
+* Access almost any filesystem (like NTFS, EXT, XFS, QNX6)
+* **Parse logs**, registry entries, cookies, history and more, directly from the source
+* Export findings to text, JSON, CSV or stream to datastores like Splunk, Elastic, MySQL
+* Feed, filter and script to process found data in any way you like
+* **Investigate images** with powerful tools like target-shell, target-mount or using our friendly Python API
 
-.. code-block:: console
+Using these features, you can for instance (among other things):
 
-    $ pip install dissect.ntfs dissect.regf
+* Make a timeline of the event log from all your investigation data at once.
+* Identify anomalies in artefacts such as services, tasks and run keys over a large amount of investigation data.
+* Do **incident response** on ransomwared (encrypted) virtual machine disks.
```

Comment:
```suggestion
* Do **incident response** on virtual machine disks encrypted by ransomware.
```
Nice example and indeed Dissect can be used for this, but unlike the other examples it is not really by default possible and needs additional programming.

Reply:
I have to admit that I did not add this point myself, but I think it is good to highlight some of the more advanced possibilities.